build-source-tarball.sh

```diff
@@ -2,22 +2,50 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-if [ -z "${1:-}" ]; then
+usage() {
     echo "usage: $0 <path-to-tarball-root> [--skip-build]"
+    echo ""
+}
+
+if [ -z "${1:-}" ]; then
+    usage
     exit 1
 fi
 
+TARBALL_ROOT=$1
+shift
+
 SKIP_BUILD=0
 
-if [ "${2:-}" == "--skip-build" ]; then
-    SKIP_BUILD=1
-fi
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
+        -?|-h|--help)
+            usage
+            exit 1
```

Comment:
AFAIK most programs exit with a 0 code on `--help`.

Reply:
Makes sense, I copied this from smoke-test.sh. Fixing.